### PR TITLE
Splint split

### PR DESCRIPTION
**Pull request by chaosvolt**
- Makes splints use ARM_EITHER and LEG_EITHER, as a look at the source (and something Coolthulhu mentioned) suggests that it will work as intended that way. What I need is to ensure ARM_EITHER actually works, no other items use it.
- Altered leg splint recipe. Numbers seem okay overall, so just a small consistency tweak.

### CODE REVIEW

**Comment by illi-kun** [comment]
Nice change!

File: data/json/recipes/recipe_others.json [hunk 1]
```diff
@@ -3252,7 +3252,7 @@
       [ "stick", 2 ],
       [ "broom", 2 ],
       [ "mop", 2 ],
-      [ "2x4", 1 ],
+      [ "2x4", 2 ],
```
Comment (BevapDin) [inline data/json/recipes/recipe_others.json]: So, it takes only half as many two by fours, but it still takes 2 brooms, sticks, mops or cues? What exactly is the logic here? (I can just craft a stick into a two by four and use it instead). And if it takes less wood, shouldn't it take less of the other other stuff as well?

If it takes less components, the item should be smaller and have less weight.

Comment (chaosvolt) [inline data/json/recipes/recipe_others.json]: That's the ORIGINAL behavior, so you should be asking that question to whoever did that. I changed it from requiring a single 2x4 to needing 2. >_>

Comment (BevapDin) [inline data/json/recipes/recipe_others.json]: You're right, my fault. Apparently colors confuse me.

**Pull request merged**